### PR TITLE
reSTify PEP 275

### PR DESCRIPTION
**Pull request by mozillazg**
refer #4

### CODE REVIEW

File: pep-0275.txt [hunk 1]
```diff
@@ -37,312 +41,327 @@ Problem
...
-                else:
-                    SUITE
-
-            when EXPR:
-                in CONSTANT_TUPLE:
-                    SUITE
-                in CONSTANT_TUPLE:
-                    SUITE
-                ...
+New Syntax::
```
Comment (brettcannon) [inline pep-0275.txt]: Sub-section?

File: pep-0275.txt [hunk 2]
```diff
@@ -37,312 +41,327 @@ Problem
...
+none of the defined cases matches, no action is taken and
+the switch statement is ignored. This is in line with the
+current if-behaviour. A user who wants to signal this
+situation using an exception can define an else-branch
+which then implements the intended action.
+
+Note that the constants need not be all of the same type, but
+they should be comparable to the type of the switch variable.
+
+Implementation:
```
Comment (brettcannon) [inline pep-0275.txt]: Sub-section

File: pep-0275.txt [hunk 3]
```diff
@@ -37,312 +41,327 @@ Problem
...
+    >>43  LOAD_CONST        0 (None)
+      46  RETURN_VALUE
+
+Where the 'SWITCH' opcode would jump to 14, 22, 30 or 38
+depending on 'x'.
+
+Thomas Wouters has written a patch which demonstrates the
+above. You can download it from [1]_.
+
+Issues:
```
Comment (brettcannon) [inline pep-0275.txt]: Sub-section

**Comment by mozillazg** [comment]
@brettcannon Thanks for your review! Please check again.

**Pull request merged**